.docker/cypress/spec/cypress/support/constants.js

```diff
@@ -0,0 +1,22 @@
+export const root_user = {
+  login: 'hitobito-pbs@puzzle.ch',
```

Comment:
Detail: für mich besteht ein Login aus Username und Passwort, was dann so aussehen würde:

```javascript
export const root_login = {
   username: 'hitobito-pbs@puzzle.ch',
   password: 'hito42bito'
}
```

Ich versuche jeweils auch Abkürzungen in Variablen zu vermeiden, dank autocompletion muss man heutzutage ja nicht mehr tippen 😉